apps/api_accounts/test/api_accounts_test.exs

```diff
@@ -123,6 +176,34 @@ defmodule ApiAccountsTest do
       assert ApiAccounts.list_keys_for_user(user) == []
     end
 
+    test "delete_tenable_user/1 deletes the user wasscan tenable" do
+      valid_users = list_of_valid_users()
+      valid_and_invalid_users = add_invalid_users()
+      assert ApiAccounts.list_users() == valid_and_invalid_users
+
+      for user <- valid_and_invalid_users do
+        for _ <- 1..5 do
+          {:ok, _} = ApiAccounts.create_key(user)
+        end
+      end
+
+      assert ApiAccounts.delete_tenable_users() == :ok
+      assert ApiAccounts.list_users() == valid_users
+
+      invalid_users =
+        Enum.map(valid_and_invalid_users, fn x ->
```

Comment:
Since this code isn't getting merged, you don't have to fix it, but just FYI, what would be better here is `Enum.filter`. If you want to go through a list and reduce its size by some criterion, that's a filter; if you want to go through a list and return a new list _of the same size_ but with every element transformed, that's a map.

So this could be:

```elixir
valid_and_invalid_users
|> Enum.filter(fn x -> x.email =~ ~r/wasscan.*@tenable.com/ end)
|> Enum.each(fn user -> 
  assert ApiAccounts.list_keys_for_user(user) == []
end)
```